pyanaconda/payload/dnfpayload.py

```diff
@@ -446,6 +446,16 @@ def add_repo(self, ksrepo):
         self._fetch_md(ksrepo.name)
         super().add_repo(ksrepo)
 
+    def add_disabled_repo(self, ksrepo):
+        """Add a disabled repo to dnf and kickstart repo lists.
+
+        :param ksrepo: Kickstart Repository to add
+        :type ksrepo: Kickstart RepoData object.
+        :returns: None
+        """
+        ksrepo.enabled = False
+        super().add_disabled_repo(ksrepo)
+
```

Comment:
In this case you can remove this dnfpayload code completely. It will call the same code from the `payload/__init__.py` so there is no reason to overload this method.

The only problem is that the repo won't be checked on validity before it is enabled but that is I guess even benefit because it won't slow down the installation in case the user don't want to use that repository.

Reply:
I'll have it fixed shortly.